src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-router.ts

```diff
@@ -12,5 +12,5 @@ JiraConnectEnterpriseAppRouter.post("/", JiraContextJwtTokenMiddleware, JiraConn
 
 JiraConnectEnterpriseAppRouter.route("/:uuid")
 	.get(csrfMiddleware, JiraJwtTokenMiddleware, JiraConnectEnterpriseAppCreateOrEdit)
-	.put(csrfMiddleware, JiraContextJwtTokenMiddleware, JiraConnectEnterpriseAppPut)
+	.put(JiraContextJwtTokenMiddleware, JiraConnectEnterpriseAppPut)
```

Comment:
Again, the csrf middleware only works in GET. We need to investigate this separately.